lib/std/Build.zig

```diff
@@ -1596,6 +1598,45 @@ pub const Module = struct {
     /// file of directory of files which constitute the module.
     source_file: FileSource,
     dependencies: std.StringArrayHashMap(*Module),
+    include_dirs: std.ArrayListUnmanaged(CompileStep.IncludeDir) = .{},
+    lib_paths: std.ArrayListUnmanaged([]const u8) = .{},
+    system_libs: std.ArrayListUnmanaged([]const u8) = .{},
+    libs: std.ArrayListUnmanaged(*Build.CompileStep) = .{},
+    config_headers: std.ArrayListUnmanaged(*Build.ConfigHeaderStep) = .{},
```

Comment:
It's probably fine to use managed containers here, that's what we normally do in `std.Build`; we aren't trying to be overly efficient with memory consumption in the build system since it doesn't reaaaaally matter.

Reply:
Ok. I originally used unmanaged containers because it makes default initialization easier, but I've swapped to managed containers and added an init function just to make things a bit cleaner.